packages/azure-functions/ProcessApplication/__tests__/index.spec.js

```diff
@@ -101,6 +108,47 @@ describe('Processing an application', () => {
         expect(context.bindings.outputSbQueue).toEqual(req.body)
         expect(context.bindings.outputSbQueue.landownerGainSiteRegistration.gainSiteReference).toEqual('test')
         expect(dbQueries.createApplicationReference.mock.calls).toHaveLength(0)
+        expect(dbQueries.deleteApplicationSession.mock.calls).toHaveLength(1)
```

Comment:
Does this also need to include a check on an expected call to getApplicationStatus?